src/App.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+enum DefaultPageValues {
+  startPage = 1,
+  defaultPageSize = 5,
+  totalPages = 42,
+}
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(DefaultPageValues.startPage);
+  const [itemsPerPage, setPerPage] = useState(
+    DefaultPageValues.defaultPageSize,
+  );
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${currentPage} (items ${currentPage * itemsPerPage - itemsPerPage + 1} - ${currentPage * itemsPerPage > DefaultPageValues.totalPages
+          ? DefaultPageValues.totalPages
+          : currentPage * itemsPerPage} of ${DefaultPageValues.totalPages})`}
```

Comment:
Such a code cannot be read. It is necessary to transfer all calculations to separate variables and use the finished result.